plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/sdk/AppEngineSDKClasspathContainer.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.sdk;
```

Comment:
I'm not sure a new package is called for here. Any reason not to just put it in the facet pckage?